src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -143,6 +143,21 @@ public void testDefaultAesCipherParams() throws Exception {
         assertEncryptionUsesExpectedDefaultParams(plaintextLength136);
     }
 
+    /**
+    * verifying encrypted plaintext is same for default AlgorithmParams and with specified AlgorithmParams.
+    * @param plaintext to be encrypted by both specifying and default AlgorithmParams.
+    */
+    private static void assertEncryptionUsesExpectedDefaultParams(String plaintext) throws Exception {
+        String actualCiphertext = encryptWithoutSpecifyingAlgorithmParams(plaintext);
+        String expectedCiphertext = StringHelper.encrypt(plaintext);
+        assertEquals(expectedCiphertext, actualCiphertext);
+    }
+
+    /**
+     * Encrypts plaintext without specifying AlgorithmParams.
+     * @param plaintext to be encrypted.
+     * @return encrypted hexadecimal string.
```

Comment:
```java
* Encrypts plaintext without specifying mode and padding scheme during @link{Cipher} initialization.
*
* @param plaintext the plaintext to encrypt as a hexadecimal string
* @return ciphertext the ciphertext as a hexadecimal string.
```

Note the blank line between the method's short description and @param. Might want to check the syntax for @link in case its wrong - can't remember off the top of my head.